### ISSUE TITLE
Feature Request: Paging

### ISSUE THREAD

**Issue by emouawad** [custom-cards/list-card#4]
This is a great Card - thanks!
Would be possible to add an AutoPaging property - therefore row_limit: x would become perhaps page_limit? I guess would be a nice addition ;) thanks :D

**Comment by iantrich** [comment]
Not sure I follow exactly. Are you asking for a paginator?

**Comment by emouawad** [comment]
yes - a client side paginator - that way long lists won't take the whole page.

**Comment by iantrich** [comment]
That's a great idea. Not sure when I'll have time to start looking into it, but I do like this idea

**Comment by iantrich** [comment]
Possible starting point: https://www.webcomponents.org/element/david-mulder/paper-datatable

**Comment by emouawad** [comment]
Excellent! can't wait 😁

**Comment by emouawad** [comment]
Any Progress on this?

**Comment by iantrich** [comment]
No. Not a high priority for me at the moment. Welcome a PR for it though 😃
